app/src/test/java/org/oppia/app/testing/TopicRevisionSpanTest.kt

```diff
@@ -0,0 +1,83 @@
+package org.oppia.app.testing
+
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.intent.Intents
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth
+import com.google.firebase.FirebaseApp
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.topic.revision.TopicRevisionFragment
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+class TopicRevisionSpanTest {
+
+  @Before
+  @ExperimentalCoroutinesApi
+  fun setUp() {
+    Intents.init()
+    FirebaseApp.initializeApp(ApplicationProvider.getApplicationContext())
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  private fun getRevisionSpanCount(activity: TopicRevisionTestActivity): Int {
+    val revisionFragment = activity
+      .supportFragmentManager
+      .findFragmentByTag(TopicRevisionFragment.TOPIC_REVISION_FRAGMENT_TAG)
+    val recyclerviewTag = activity.resources.getString(R.string.topic_revision_recyclerview_tag)
+    val recyclerView =
+      revisionFragment?.view?.findViewWithTag<RecyclerView>(recyclerviewTag)
+    return (recyclerView?.layoutManager as GridLayoutManager).spanCount
+  }
+
+  @Test
+  fun testTopicRevisionFragmentRecyclerView_hasCorrectSpanCount() {
+    launch(TopicRevisionTestActivity::class.java).use { scenario ->
+      scenario.onActivity { activity ->
+        Truth.assertThat(getRevisionSpanCount(activity)).isEqualTo(2)
```

Comment:
Nit:
`assertThat(getRevisionSpanCount(activity)).isEqualTo(2)`

Here and below, and optimise imports accordingly.

Reply:
Done